go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/containers/podman/v5
 
-// Warning: Ensure the "go" and "toolchain" versions match exactly to prevent unwanted auto-updates
+// Warning: Ensure the "go" and "toolchain" versions match exactly to prevent unwanted auto-updates.
+// That generally means there should be no toolchain directive present.
```

Comment:
This wording confuses me a little: I might spend myself looking for a `toolchain` line. Would it make sense to rephrase as something closer to
> // Warning: if there is a "toolchain" directive anywhere in this file (and most of the time there shouldn't be), its version must be an exact match to the "go" directive

Reply:
sure